src/main/java/io/vertx/core/eventbus/impl/clustered/ClusteredHandlerHolder.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2011-2019 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.eventbus.impl.clustered;
+
+import io.vertx.core.eventbus.impl.HandlerHolder;
+import io.vertx.core.eventbus.impl.HandlerRegistration;
+import io.vertx.core.impl.ContextInternal;
+
+public class ClusteredHandlerHolder<T> extends HandlerHolder<T> {
```

Comment:
is this class really useful ?

Reply:
It allows to identify each `handlerHolder`. In the end this identifier goes to the cluster manager. In standalone mode the identifier is not useful. We could use a single class though (value would always be zero in standalone mode)